cloud

```diff
@@ -627,6 +627,16 @@ function cloud_functions_deploy() {
                           --trigger-topic "${trigger_topic}" \
                           --memory 256MB \
                           --timeout 540
+    cloud_function_deploy "$project" "${prefix}pick_notification" \
+                          "$source" \
+                          --entry-point kcidb_pick_notification \
+                          --env-vars-file "$env_yaml" \
+                          --runtime python37 \
+                          --trigger-event "${trigger_event}" \
+                          --trigger-resource "${trigger_resource}" \
+                          --memory 256MB \
+                          --retry \
+                          --timeout 540
```

Comment:
This will make `kcidb_pick_notification` trigger off new notifications getting created, but we need to trigger it off messages from your cloud scheduler job arriving to the message queue topic you would create for it.

Start with copying the `cloud_function_deploy` invocation for `kcidb_load_queue` function.
Copy and modify creation of the `kcidb_trigger` topic to create your own.
Copy and modify creation of the `kcidb_trigger` Cloud Scheduler job to create your own.